T281: Add https support to the load command

c-po opened the pull request.

c-po committed:
T281: Add https support to the load command

UnicronNL commented:
c-po,
What about this?
https://github.com/vyos/vyatta-cfg/pull/9/files#diff-dc9ca7ee8c0d354c8d868e682a2876a1L106
And also what about ssl

c-po committed:
T281: Validate error codes for 'http' and 'https'

c-po committed:
T281: Support 'http' permanent redirects (code 301)

This enables fetching a VyOS config from a http://foo/bar URL even when
http://foo/bar is a permanent redirect to https://foo/bar.

c-po commented:
@UnicronNL you are correct. ALso added support for permanent http redirects (code 301)

c-po committed:
Merge remote-tracking branch 'upstream/current' into add-load-https-support

* upstream/current:
  Revert "T441: Remove dependency on vyatta-quagga. This package should not depend on any routing stack anyway."
